demisto_sdk/generate_docs/generate_script_doc.py

```diff
@@ -0,0 +1,204 @@
+    :param script_path: the script yml file path.
+    :return: list of dicts contains the script information.
+    """
+    script = get_yaml(script_path)
+    script_type = script.get('subtype')
+    if not script_type:
+        script_type = script.get('type')
+
+    tags = script.get('tags')
+    tags = ', '.join(map(str, tags))
+
+    docker_images = get_docker_images(script)
+    docker_images = ', '.join(map(str, docker_images))
+    from_version = get_from_version(script_path)
+
+    return [{'Name': 'Script Type', 'Description': script_type},
+            {'Name': 'Tags', 'Description': tags},
+            {'Name': 'Docker Image', 'Description': docker_images},
+            {'Name': 'Demisto Version', 'Description': from_version}]
+
+
+def get_inputs(script):
+    """
+    Gets script inputs.
+    :param script: the script object.
+    :return: list of inputs and list of errors.
+    """
+    errors = []
+    inputs = []
+
+    if not script.get('args'):
+        return {}, []
+
+    for arg in script.get('args'):
+        if not arg.get('description'):
+            errors.append(
+                'Error! You are missing description in script input {}'.format(arg.get('name')))
+
+        inputs.append({
+            'Argument Name': arg.get('name'),
+            'Description': stringEscapeMD(arg.get('description', ''))
+        })
+
+    return inputs, errors
+
+
+def get_outputs(script):
+    """
+    Gets script outputs.
+    :param script: the script object.
+    :return: list of outputs and list of errors.
+    """
+    errors = []
+    outputs = []
+
+    if not script.get('outputs'):
+        return {}, []
+
+    for arg in script.get('outputs'):
+        if not arg.get('description'):
+            errors.append(
+                'Error! You are missing description in script output {}'.format(arg.get('contextPath')))
+
+        outputs.append({
+            'Path': arg.get('contextPath'),
+            'Description': stringEscapeMD(arg.get('description', '')),
+            'Type': arg.get('type', 'Unknown')
+        })
+
+    return outputs, errors
+
+
+def get_used_in(id_set_path, script_id):
+    """
+    Gets the integrations, scripts and playbooks that used the input script, without test playbooks.
+    :param id_set_path: updated id_set.json file path.
+    :param script_id: the script id.
+    :return: list of integrations, scripts and playbooks that used the input script
+    """
+    id_set = get_json(id_set_path)
+    used_in_list = []
+
+    id_set_sections = list(id_set.keys())
+    id_set_sections.remove('TestPlaybooks')
+
+    for key in id_set_sections:
+        items = id_set[key]
+        for item in items:
+            key = list(item.keys())[0]
+            scripts = item[key].get('implementing_scripts', [])
+            if scripts and script_id in scripts:
+                used_in_list.append(item[key].get('name', []))
+    return list(filter(None, dict.fromkeys(used_in_list)))
```

Comment:
Why do you need it?

Reply:
To delete duplicates from the list, i replaced it to set.